r/R/install-arrow.R

```diff
@@ -51,14 +51,21 @@ install_arrow <- function(nightly = FALSE,
                           minimal = Sys.getenv("LIBARROW_MINIMAL", FALSE),
                           repos = getOption("repos"),
                           ...) {
-  if (tolower(Sys.info()[["sysname"]]) %in% c("windows", "darwin", "linux")) {
-    Sys.setenv(
-      LIBARROW_DOWNLOAD = "true",
-      LIBARROW_BINARY = binary,
-      LIBARRWOW_MINIMAL = minimal,
-      ARROW_USE_PKG_CONFIG = use_system
-    )
-    install.packages("arrow", repos = arrow_repos(repos, nightly), ...)
+  sysname <- tolower(Sys.info()[["sysname"]])
+  conda <- isTRUE(grepl("conda", R.Version()$platform))
+
+  if (sysname %in% c("windows", "darwin", "linux")) {
+    if (conda && !nightly && (sysname %in% c("darwin", "linux"))) {
```

Comment:
Why not Windows?

Reply:
AFAIU (reading the language on the README), we aren't making conda package for windows for R. @xhochy is that accurate? I haven't gotten involved with the conda packaging so I don't have first-hand knowledge.

Reply:
Based on https://anaconda.org/conda-forge/r-arrow, that seems correct (only linux and osx packages)

Reply:
Yes, would be glad to have some help there. Last time I tried a windows build it failed with problem unknown to me.